src/ducks/Watchlists/Actions/Edit/EditForm/Assets.js

```diff
@@ -0,0 +1,193 @@
+import React, { useState, useRef, useEffect, useMemo } from 'react'
+import cx from 'classnames'
+import Label from '@santiment-network/ui/Label'
+import Icon from '@santiment-network/ui/Icon'
+import Input from '@santiment-network/ui/Input'
+import Panel from '@santiment-network/ui/Panel'
+import { Checkbox } from '@santiment-network/ui/Checkboxes/Checkboxes'
+import { useTheme } from '../../../../../stores/ui/theme'
+import { useAllProjects, useOnClickOutside } from './hooks'
+import styles from './index.module.scss'
+import cardStyles from '../../../../../ducks/Watchlists/Widgets/Table/AssetCard.module.scss'
+import fieldStyles from '../../../../../ducks/Studio/Sidebar/ProjectSelector/index.module.scss'
+
+const VIEW_ITEM_COUNT = 4
+
+const Assets = ({ watchlist, onChange }) => {
+  const ref = useRef()
+  const [isSearchMode, setIsSearchMode] = useState(false)
+  const [filter, setFilter] = useState('')
+  const watchListItems = useMemo(
+    () => (watchlist ? watchlist.listItems.map(l => l.project) : []),
+    [watchlist]
+  )
+  const [items, setItems] = useState(watchListItems)
```

Comment:
It's not clear how it's used from the variable name. If it's filtered projects (assumption based on the `LOC 52-61`) better to name it `filteredProjects`